### PR TITLE
Design Teardown

### PR DESCRIPTION
**Pull request by adejam**
# Project: DESIGN TEARDOWN
The main objective of this project is to build a grayscale heat map of [Smashing Magazine Homepage](https://www.smashingmagazine.com/) that indicates which elements have the greatest weight in the visual hierarchy using HTML and CSS.

The PR does the following:
- Follows the guidelines on [Odin project assignment page](https://www.theodinproject.com/courses/html5-and-css3/lessons/design-teardown) to build a greyscale heat map of [Smashing Magazine Homepage](https://www.smashingmagazine.com/).

- Uses CSS flexbox and float for positioning
- Uses Semantic html
- Uses CSS Arial and Sans-serif Font Family
- Uses stylint and stickler for linter 
- [Live Demo Link](https://raw.githack.com/adejam/design-teardown/page-design/index.html)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,96 @@
 # Design Teardown
-The main objective of this project is to build a design teardown of [Smashing Magazine Homepage](https://www.smashingmagazine.com/) using HTML and CSS.
+The main objective of this project is to build a grayscale heat map of [Smashing Magazine Homepage](https://www.smashingmagazine.com/) that indicates which elements have the greatest weight in the visual hierarchy using HTML and CSS.
 
+The project uses semantic html and CSS positioning and floating elements. CSS flexbox in conjunction with CSS float property is used for effective positioning of the elements.CSS Arial and Sans-serif Font Family were also used.
 
+This project demonstrates visual hierarchy, typography and design principles.
```
Comment (bolah2009) [inline README.md]: I think it's worth noting that the reason you using `grayscale heat map` is to demonstrate  `visual hierarchy`

This `CSS Arial and Sans-serif Font Family were also used` should also be associated with the demonstration of `typography` in the project.

File: stylesheet.css [hunk 2]
```diff
@@ -0,0 +1,326 @@
...
+}
+
+.time {
+  margin-bottom: 0.5em;
+  font-size: 0.8em;
+}
+
+.sub_article_h2 {
+  margin-bottom: 1em;
+}
```
Comment (bolah2009) [inline stylesheet.css]: Please take care of this, seems you repeating `margin-bottom: 1em;` unnecessarily.

File: stylesheet.css [hunk 3]
```diff
@@ -0,0 +1,326 @@
...
+  margin: 10px;
+}
+
+.footer {
+  padding: 50px;
+}
+
+.footer_h {
+  margin: 30px;
+}
```
Comment (bolah2009) [inline stylesheet.css]: Seems a class already have a `margin: 30px;` property, let's avoid repetition, please.

File: stylesheet.css [hunk 4]
```diff
@@ -0,0 +1,326 @@
...
+body,
+h1,
+h2,
+a,
+p,
+h3,
+h4,
+h5,
+h6,
+li,
```
Comment (bolah2009) [inline stylesheet.css:22]: I think `li` should have a `list-style-type: none;` property. See below:

src-2

Untitled




I really don’t think this should have the dots... 🤔

**Review by bolah2009 — APPROVED** [review]
Project Looks Good To Me!

Let's fix the issues raised before merging!

File: README.md (hunk 1 above)
Comment (adejam) [reply README.md]: Thanks for the review. This will be improved on.

File: stylesheet.css (hunk 2 above)
Comment (adejam) [reply stylesheet.css]: Thanks for the review. I'll work on this.

File: stylesheet.css (hunk 4 above)
Comment (adejam) [reply stylesheet.css:22]: Thanks for the review. Most list in the project has ```list-style-type: bullet```. I created a class to set ```list-style-type: none```, i'll add it to the list in the navbar to correct the ```list-style-type``` here .

Comment (bolah2009) [reply stylesheet.css:22]: Okay 👍🏾

**Pull request merged**